src/vs/editor/contrib/codeAction/browser/codeActionCommands.ts

```diff
@@ -491,7 +491,7 @@ export class AutoFixAction extends EditorAction {
 	}
 }
 
-const weight = KeybindingWeight.EditorContrib + 90;
+const weight = KeybindingWeight.EditorContrib + 1000;
```

Comment:
Not ideal. The issue is that in notebooks `list.clear` ends up eating `escape` otherwise. Not sure the best way to disable this